src/clusterfuzz/_internal/bot/tasks/trials.py

```diff
@@ -25,11 +25,25 @@
 TRIALS_CONFIG_FILENAME = 'clusterfuzz_trials_config.json'
 
 
+class AppArgs:
+
+  def __init__(self, probability, contradicts):
+    self.probability = probability
+    self.contradicts = contradicts
+
+
+class AppArgs:
```

Comment:
Twice?

Reply:
this was a rebasing mistake, now it is fixed